kubernetes/kubernetes
# remove usage of the test/test_owners.* files

**What this PR does / why we need it**:

- remove update_owners.py; seems responsible for generating
test_owners files
- remove the existing test/test_owners.* files
- remove verify-test-owners.sh and references to it

**Which issue(s) this PR fixes**:

xref https://github.com/kubernetes/test-infra/pull/14845
https://github.com/kubernetes/test-infra/issues/14839

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:


```docs

```

/kind cleanup
/priority backlog
/sig testing
